ios/Classes/Constants/ArgumentKey.m

```diff
@@ -23,7 +23,7 @@ @implementation ArgumentKey
 NSString * const ARGUMENT_KEY_SERVICE_ID = @"serviceId";
 NSString * const ARGUMENT_KEY_CHARACTERISTIC_UUID = @"characteristicUuid";
 NSString * const ARGUMENT_KEY_CHARACTERISTIC_IDENTIFIER = @"characteristicIdentifier";
-NSString * const ARGUMENT_KEY_BYTES = @"bytes";
+NSString * const ARGUMENT_KEY_VALUE = @"value";
```

Comment:
are you able to test if it works?

Reply:
I'm getting `Error Domain=CBATTErrorDomain Code=3 "Writing is not permitted."` when running test with real device SensorTag but I suppose it's fine, because test scenario is written for simulated devices, right?

Reply:
Tested again - error does not occur anymore